### DISCUSSION TITLE
How to upload to azure storage container?

### DISCUSSION THREAD

**Discussion by MadalasaP** [minio/minio#16838] [Q&A] [answered]
I have been trying to upload data from minio to azure containers. 
I have tried azcopy it is accepting minio URL as a file. I'm not sure how to use the mcli alias set for azure. 
minio URL: http://192.168.6.27:9000. Could anyone help me how to upload to azure containers using either mcli cp or azcopy.

**Comment by harshavardhana** [comment, marked as answer]
You cannot use azcopy with MinIO. MinIO speaks S3 API not azure API.

**Reply by MadalasaP** [reply]
Is there another way to upload to azure containers?

**Reply by harshavardhana** [reply]
Beyond our scope here @MadalasaP
